ChineseFilter/sections/specific.txt

```diff
@@ -938,6 +938,7 @@ olevod.com##.L4A
 olevod.com##.A4R
 olevod.com##.ad1
 olevod.com##.L1
+sm.ms##.wwads_abp
```

Comment:
This is an antiadblock message, please move the rule to annoyances > antiadblock.txt

Reply:
ok. finished.